### ISSUE TITLE
ip address 159.226.126.127

### ISSUE THREAD

**Issue by ghost** [Project-Evidence/project-evidence.github.io#11]
Regarding:

> We are looking for contributors to help us find the domain name for the IP address: 159.226.126.127. It appears to be unregistered. We are confused as to how this web server was found.

that is the ip address of the `Scientific Digital Library`, its (now dead) link can be found in the lab homepage:

![library](https://raw.githubusercontent.com/davidfregoli/project-evidence.github.io/master/ip-address-159.226.126.127.png)

**Comment by Project-Evidence** [comment]
Oops, we appear to have missed this issue in [Revision 2](project-evidence.github.io/). Please feel free to open a PR to correct this. You should be looking for an @margin-note in src/sections/claim2.scrbl.

**Comment by mukeshp70** [comment]
No,IP address: 159.226.126.127 belongs to China Science and Technology Network (Abuse Contract for 159.226.0.0 to 159.226.255.255 is ipas@cnnic.cn). Confirmed from two independent sources:

[WHOIS IP Lookup Tool _ UltraTools.pdf](https://github.com/Project-Evidence/project-evidence.github.io/files/4575235/WHOIS.IP.Lookup.Tool._.UltraTools.pdf)

[IP WHOIS Lookup - Lookup IP WHOIS Information - WhatIsMyIP.com®.pdf](https://github.com/Project-Evidence/project-evidence.github.io/files/4575238/IP.WHOIS.Lookup.-.Lookup.IP.WHOIS.Information.-.WhatIsMyIP.com.pdf)

However, unable to archive in archive.is.

**Comment by mukeshp70** [comment]
Archived another at : https://archive.is/gN2yx

**Comment by research-project-cov** [comment]
it has been removed; I could have access to it in april; now only in archives of google; I kept pdf
